File: app/src/main/java/cjx/com/diary/base/BaseFragment.java
package cjx.com.diary.base;

import android.content.Context;
import android.support.v4.app.Fragment;

/**
 * @author: bear
 *
 * @Description: BaseFragments
 *
 * @date: 2017/5/10
 */
public abstract class BaseFragment extends Fragment implements BaseView {
    public BaseActivity mActivity;

    @Override
    public void onAttach(Context context) {
        super.onAttach(context);
        mActivity = (BaseActivity) context;
    }
}

File: app/src/main/java/cjx/com/diary/base/BaseFragment.kt
package cjx.com.diary.base

import android.support.v4.app.Fragment

/**
 * description:
 * author: bear .
 * Created date:  2019-06-04.
 */
open class BaseFragment : Fragment(), BaseView{
    fun mActivity(activity: BaseActivity): BaseActivity {
        return activity
    }
}

